Files/Files.csproj

```diff
@@ -986,28 +986,28 @@
       <Version>7.0.0-preview4</Version>
```

Comment:
You need to update all the packages together.

Reply:
Resolved with bc21392c73043e9e98d9608bf9aaa7a8d0d83c55.